### PR TITLE
CON-1176: Review Cloud Docs

### PR DESCRIPTION
**Pull request by JohnJoser3**
I have edited all the docs in this repository as per the findings in the [content audit](https://jor3cev-my.sharepoint.com/:x:/r/personal/john_jose_r3_com/_layouts/15/Doc.aspx?sourcedoc=%7B8672C637-DFE0-49F5-B3FA-9F2BDCB49624%7D&file=Conclave-%20Content%20Audit%20v2.0.xlsb.xlsx&action=default&mobileredirect=true&DefaultItemOpen=1&login_hint=john.jose%40r3.com&ct=1666335906647&wdOrigin=OFFICECOM-WEB.START.REC&cid=b12895d2-2979-4eba-9f4a-55834b6427d5). Main focus of this set of changes is to improve clarity, simplicity, and usability of the docs. Also, I have corrected a few errors and updated outdated references. The PR reference number CON-1176 refers to [this ticket](https://r3-cev.atlassian.net/browse/CON-1176?atlOrigin=eyJpIjoiNWRkZDhmMmZjMGY2NGIxMGI2M2YwOTJhYThlNmQwZTgiLCJwIjoiaiJ9) raised in the Conclave Core Jira board.

### CODE REVIEW

File: user-guide/docs/conclavepass-sample.md [hunk 1]
```diff
@@ -1,223 +1,200 @@
...
-persist the user databases. The functions component uses the JavaScript
-`fetch()` built-in capability to query and update an external data store with
-each user's encrypted database entries. It is vitally important to ensure that
-any data exchanged via `fetch()` is encrypted as the request is made outside the
-Conclave Functions Intel SGX enclave. The functions module uses another built-in
-function, `crypto.getProjectKey()` to get a key that is unique to the project and
-function code and uses this to encrypt/decrypt the user's database prior to
-exchanging it with the external storage.
+
+The functions component consists of a typescript that provides methods that you can use to add, query, get,
```
Comment (roy-hopkins) [inline user-guide/docs/conclavepass-sample.md:47]: "a typescript that..." doesn't read correctly. Maybe "a typescript module that..."

File: user-guide/docs/conclavepass-sample.md [hunk 2]
```diff
@@ -1,223 +1,200 @@
...
+You must host each of the below components to run the demonstration.
 
 ### 1. Backend
-The backend service needs to be available as a fixed IP on the internet in order
-to allow Conclave Functions to access it for storing user databases.
 
-It is recommended to set up a new virtual machine with your cloud service
-provider to host this. The resource requirements are tiny so you can use the
-smallest size of virtual machine your cloud service provider provides.
+The backend service needs to be available as a fixed IP on the internet to allow Conclave Functions to access it for
```
Comment (roy-hopkins) [inline user-guide/docs/conclavepass-sample.md:126]: "available with a fixed IP address on the internet."

File: user-guide/docs/conclavepass-sample.md [hunk 3]
```diff
@@ -1,223 +1,200 @@
...
 
-It is recommended to set up a new virtual machine with your cloud service
-provider to host this. The resource requirements are tiny so you can use the
-smallest size of virtual machine your cloud service provider provides.
+The backend service needs to be available as a fixed IP on the internet to allow Conclave Functions to access it for 
+storing user databases.
 
-The service is built using this command:
+Conclave Cloud recommends setting up a new virtual machine with your cloud service provider to host this. You can 
+choose a VM with the smallest size, as the resource requirements are less.
```
Comment (roy-hopkins) [inline user-guide/docs/conclavepass-sample.md:130]: "the resource requirements are minimal".

File: user-guide/docs/index.md [hunk 4]
```diff
@@ -2,100 +2,82 @@
...
+Once you have signed up, you can log in to the Conclave Cloud portal.
 
 ## What next?
 
-From here you need to create yourself a project and then upload one or more
-functions to play with.
+You need to create a project and upload one or more functions to your project.
 
-Use the resources below to guide you through using the platform.
+Use the resources below to use the platform.
```
Comment (roy-hopkins) [inline user-guide/docs/index.md:24]: This doesn't read clearly with 'use' twice.

File: user-guide/docs/index.md [hunk 5]
```diff
@@ -2,100 +2,82 @@
...
+You need Java 11 or higher runtime environment to run the `ccl` tool.
 
-_Linux:_
+=== "Linux/macOS"
 
-1. Download the tool using the link above.
-2. Copy the tool into a directory that is on your path (such as `~/bin`) _or_
-   add the directory containing the tool to your `PATH` environment variable.
-3. Test it by executing `ccl`.
+    1. Download the `ccl` tool using [this link](https://github.com/R3Conclave/ccl-documentation/releases/download/1.0.0-beta2/ccl).
```
Comment (roy-hopkins) [inline user-guide/docs/index.md:56]: Can we add another step here?
`chmod +x ccl` to make the tool executable.

File: user-guide/docs/index.md [hunk 6]
```diff
@@ -105,27 +87,27 @@ correct path to where you unzipped the SDK.
...
 
-For the beta version of Conclave Cloud, the Java/Kotlin Client SDK is delivered
-in a zip file that contains a file-based maven repository. This can be added as
-a dependency in your Java/Kotlin project by adding the repository then
-describing the dependency.
 
-Download and unzip the SDK to the directory of your choice. Add the repository
-to your `build.gradle.kts`, making sure to provide the correct path to where you
-unzipped the SDK.
+1. Download and unzip [the SDK](https://github.com/R3Conclave/ccl-documentation/releases/download/1.0.0-beta2/conclave-cloud-sdk-java-1.0.0-beta2.zip)
```
Comment (roy-hopkins) [inline user-guide/docs/index.md:94]: I think this link is now out of date - shouldn't be `beta2`. 
https://github.com/R3Conclave/ccl-documentation/releases/download/1.0.0-GA/conclave-cloud-sdk-java-1.0.0-GA.zip

File: user-guide/docs/what-is-conclave-cloud.md [hunk 7]
```diff
@@ -1,91 +1,67 @@
...
+frontend, a Kotlin-based CLI tool, and a Spring backend for persistence. You can find the sample in
+[this GitHub repository](https://github.com/R3Conclave/ccl-sample-conclavepass).
+
+## Future plans of the Conclave Cloud platform
+
+The Conclave Cloud platform will bring together an expanding set of services that seamlessly integrate with each 
+other, providing a rich set of tools for implementing solutions without ever having to leave the Conclave Cloud 
+platform.
+
+![An architecture diagram of the proposed Conclave CLoud ecosystem](assets/ImmersiveEcosystem-CCL.drawio.png)
```
Comment (roy-hopkins) [inline user-guide/docs/what-is-conclave-cloud.md:65]: Capital L in "Conclave CLoud" in the link name.

**Review by roy-hopkins — CHANGES_REQUESTED** [review]
Really good changes! I've highlighted a couple of things to address, otherwise it looks fantastic!

**Comment by JohnJoser3** [comment]
Fixed the review comments. Please take a look.

**Review by roy-hopkins — APPROVED** [review]
Excellent improvements! Sorry the review took so long.

**Pull request merged**